index.html

```diff
@@ -4,18 +4,169 @@
 <head>
     <meta charset="UTF-8">
 
-    <title>Untitled</title>
+    <title>Smashing Magazine - HeatMap</title>
 
     <link rel="stylesheet" href="css/styles.css">
 
+    <link href="https://fonts.googleapis.com/css?family=Fira+Sans:200,500,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
     <header>
-
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont "></div>
     </header>
 
     <main>
+        <div class="title-div">
```

Comment:
Here the observation is the following:
* [ ] Every child of main should be a `<section>` HTML tag pair. **So every this part of the site should be encapsulated in within those tags.**
* [ ] Every child of section should be an `<article>` HTML tag pair. **And instead of using `<div>` we should use `<article>` tags but you might use a div as a wrapper if you think is necessary but the main content should have an `<article>` pair tags**

Also for this section, we should follow these instructions:
* [ ] Four articles with a width of `50%`
* [ ] Each container should have an image in the top left corner with the title of an article under it.
**As you can see from the original site, there are four articles up there instead of the three you made we should follow the same guidelines provided before and have four articles with a width of 50% and replicate in some manner the way the articles are presented on the original site**